search-app/src/cmr/search/api/providers.clj

```diff
@@ -0,0 +1,54 @@
+(ns cmr.search.api.providers
+  "Defines the API for search-by-concept in the CMR."
```

Comment:
Update the docstring to what it really does.

Reply:
Done.